apis/v1beta1/gateway_types.go

```diff
@@ -70,13 +70,20 @@ type GatewaySpec struct {
 	// At least one Listener MUST be specified.
 	//
 	// Each listener in a Gateway must have a unique combination of Hostname,
-	// Port, and Protocol. Below combinations are considered Core and MUST be
-	// supported:
+	// Port, and Protocol.
+	//
+	// Within the HTTP Conformance Profile, the below combinations of port and
+	// protocol are considered Core and MUST be supported:
 	//
 	// 1. Port: 80, Protocol: HTTP
 	// 2. Port: 443, Protocol: HTTPS
 	//
-	// Port and protocol combinations not in this list are considered Extended.
+	// Within the TLS Conformance Profile, the below combinations of port and
+	// protocol are considered Core and MUST be supported:
+	//
+	// 1. Port: 443, Protocol: TLS
+	//
+	// Port and protocol combinations not listed above are considered Extended.
```

Comment:
Does this update make sense?

/cc @shaneutt @mlavacca

Reply:
That makes sense to me.